### ISSUE TITLE
BTT disables left click

### ISSUE THREAD

**Issue by guardingnome** [fifafu/BetterTouchTool#1858]
**IMPORTANT:** Don't post private requests here (e.g. license issues, etc.). Please send me an email at boastr.net@gmail.com instead.

---

#### Description of bug/feature request/question:
During use (browsing safari) BTT suddenly stopped registering left clicks system wide.
I tried restarting, and shutting down, neither fixed.
I wasn't changing BTT settings at the time.

Quitting BTT stops issue.

#### Affected input device (e.g. MacBook Trackpad, Magic Mouse/Trackpad, Touch Bar, etc.):
Force touch Trackpad.

#### Device information:

Type of Mac: 13" MacBook Pro late 2015
macOS version: 10.13.1
BetterTouchTool version: 2.318

#### Additional information (e.g. StackTraces, related issues, screenshots, workarounds, etc.):

If I make a shortcut to click with left 1 finger force touch, it allows me to click again!
But, it forbids me from clicking and dragging. This is kinda frustrating!

Love,
writer

---

**Note:** Before bug reporting, please make sure you have the latest version of BetterTouchTool and that you have already tried to restart your system :-). If you encounter a crash, please attach a crash log from the macOS Console.app from the "User Diagnostic Reports" section.

**Comment by fifafu** [comment]
Maybe the same as maybe the same as https://github.com/fifafu/BetterTouchTool/issues/1829?

**Comment by guardingnome** [comment]
Could be, I have my settings set quite low.
They always have been set this low, however.

I used to experience the related bug you mentioned, but my nonstop tweaking of the pressure values eventually fixed it.

I’ll try again when I get to work.

Love,

Kai

**Comment by guardingnome** [comment]
I changed all of the pressure settings, no luck.

For what it’s worth, they are not all increasing in pressure.

The four finger press is much lower than the three finger, given the way I have my gestures set up. 

But, the one is usually 30-50, two is ~90-100, three is ~200, four is ~100, five is ~300.

Love,

Kai

**Comment by fifafu** [comment]
Mhh then it's probably something else.
If possible go to the about tab in BTT and click the "export debug information" button. This will create a folder called "BetterTouchToolDebug" on your Desktop. If you send me the contents to boastr.net@gmail.com I can probably tell what's wrong.

**Comment by fifafu** [comment]
Unfortunately I haven't been able to reproduce it even with the debug data :-(
Could you try to restart your machine? (In case you haven't done that already)

**Comment by guardingnome** [comment]
I’ll do it again, I’m not afraid!

That being said, I have restarted. I have also tried quitting BTT, powering off -unplugging- and, turning back on.

If there is some way to keep my shortcuts, I’ll try reinstalling.

Thanks for the app etc-

Love,

Kai

**Comment by fifafu** [comment]
Does the leftclick stop working completely as soon as you start BTT?

**Comment by guardingnome** [comment]
Yes, left click stops registering system wide as soon as it is launched.

Left click works when I pause BTT, but left click and drag does not work until I quit BTT.

Very strange to me.

Kai

**Comment by fifafu** [comment]
Yep very weird indeed. 
Could you check whether removing the "Single Finger Force Click" from your gestures changes anything? (That's the only gesture that has recently beend added, thus maybe there is still some unknown issue with it)

**Comment by guardingnome** [comment]
I disabled it, but that was what I had to change to be able to continue using BTT.

BTT is not usable without this, as it destroys my computer experience if I cannot click on things : (

Love

**Comment by guardingnome** [comment]
Ok. Weird development.

I paused BTT, turned of my computer.
It stayed paused, but was still working when turned on.
The computer had a kernel panic this morning, while asleep, and it was still paused and working all day. When I say working, I mean even though it is paused I can use the click bindings.
When I unpaused it recently, left click stopped working.
When I paused it again, BTT stopped working, and left click still doesn’t work.
When I quit BTT, left click works similar to before.

Attached debug info
